src/main/java/nextstep/subway/domain/path/PathStrategies.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.subway.domain.path;
+
+import nextstep.subway.domain.Section;
+
+import java.util.Arrays;
+import java.util.function.Function;
+
+public enum PathStrategies {
+
+    DISTANCE("DISTANCE", section -> section.getDistance()),
+    DURATION("DURATION",section -> section.getDuration());
```

Comment:
아래와 같이 표현하는 것도 좋은 것 같아요 😄 
```suggestion
    DISTANCE("DISTANCE", Section::getDistance),
    DURATION("DURATION", Section::getDuration);
```

Reply:
넵!